cve_bin_tool/error_handler.py

```diff
@@ -140,6 +144,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     SystemExit: -2,
     FileNotFoundError: -3,
     InvalidCsvError: -4,
+    InvalidCheckerError: -4,
```

Comment:
I think we probably want InvalidCheckerError to have its own number, so I guess -16 ?  InvalidCsvError and InvalidJsonError have the same number presumably because they're very similar parsing errors, but there's no particular reason not to give every error a separate number.